src/main/java/org/apache/mesos/scheduler/plan/api/BlockInfo.java

```diff
@@ -19,44 +17,33 @@
     private final Status status;
     private final String name;
     private final String message;
-    private final Boolean hasDecisionPoint;
 
     @JsonCreator
     public static BlockInfo create(
```

Comment:
Yes.  We spoke about this.  This PR, is too big.  Will follow up.  https://mesosphere.atlassian.net/browse/INFINITY-624